src/databricks/labs/remorph/cli.py

```diff
@@ -53,7 +53,7 @@ def transpile(
         sdk_config=w.config,
     )
 
-    status = morph(config)
+    status = morph(config, w)
```

Comment:
Can you add a readme document on how to create this workspace client , if done externally with required fields.